async-stream/Cargo.toml

```diff
@@ -9,18 +9,18 @@ name = "async-stream"
 #   - Cargo.toml
 #   - README.md
 # - Create git tag
-version = "0.2.0"
+version = "0.2.1"
 edition = "2018"
 license = "MIT"
 authors = ["Carl Lerche <me@carllerche.com>"]
 description = "Asynchronous streams using async & await notation"
-documentation = "https://docs.rs/async-stream/0.2.0/async-stream"
+documentation = "https://docs.rs/async-stream/0.2.1/async-stream"
 homepage = "https://github.com/tokio-rs/async-stream"
 repository = "https://github.com/tokio-rs/async-stream"
 readme = "README.md"
 
 [dependencies]
-async-stream-impl = { version = "0.2.0", path = "../async-stream-impl" }
+async-stream-impl = { version = "0.2.1", path = "../async-stream-impl" }
```

Comment:
Do we need to remove the path dep before releasing, or is this okay now?

Reply:
This crate doesn't have the CI infrastructure to remove it... incidentally, Tokio needs to be fixed in that regards as well.